frontend/src/modules/app/layouts/SidebarRoute.tsx

```diff
@@ -1,7 +1,6 @@
 /**
- * This is the doc comment for Sidebar based Layout
- * @packageDocumentation
- * @module sidebar-layout
+ * @summary Layout with Sidebar based pages
```

Comment:
Where should we use `SidebarRoute` as opposed to `SidebarSlideRoute`? Do we just use `SidebarRoute` in standby mode, i.e. not ventilating?

Reply:
Yes, that's right, we use SidebarRoute when not ventilating.